apps/api/src/modules/user/user.repository.ts

```diff
@@ -17,4 +18,70 @@ export class UserRepository {
       where: { castingId },
     })
   }
+
+  async getUsers(params: {
+    skip?: number
+    take?: number
+    cursor?: Prisma.UserWhereUniqueInput
+    where?: Prisma.UserWhereInput
+    orderBy?: Prisma.UserOrderByWithRelationInput
+  }): Promise<User[]> {
+    return await this.prisma.user.findMany({
+      ...params,
+    })
+  }
```

Comment:
Where are we using this

Reply:
I forgot to remove it.
I was using this for the generalized findMany function , but specialized getPendingUsers would be easier for me.